### PR TITLE
docs: updated doc for how to use gitpod

### PR DESCRIPTION
**Pull request by benbenben2**
updated doc for how-to use gitpod

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]
Thank you very much for improving the documentation!

File: docs/how-to-guides/use-gitpod.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # Using Gitpod for Remote Development
+If your computer performances are restricting you to make developments, you can use [Gitpod](https://gitpod.io).
```
Comment (diivi) [inline docs/how-to-guides/use-gitpod.md:2]:
```suggestion
If your computer performance restricts you from developing, you can use [Gitpod](https://gitpod.io). 
```

File: docs/how-to-guides/use-gitpod.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
 # Using Gitpod for Remote Development
+If your computer performances are restricting you to make developments, you can use [Gitpod](https://gitpod.io). 
+Gitpod allows you to do the devs on an ephemeral environment. It is free for a maximum of 500 credits or 
+50 hours per months (https://www.gitpod.io/pricing).
 
-[Gitpod](https://gitpod.io) provides powerful ready-to-code developer environments in the cloud eliminating the friction
+Gitpod provides a powerful ready-to-code developer environments in the cloud eliminating the friction
```
Comment (diivi) [inline docs/how-to-guides/use-gitpod.md:6]:
```suggestion
Gitpod provides a robust ready-to-code developer environment in the cloud eliminating the friction
```

File: docs/how-to-guides/use-gitpod.md [hunk 3]
```diff
@@ -12,6 +17,13 @@ in the [quickstart guide](../introduction/dev-environment-quick-start-guide.md)
 and [docker-developer-guide](docker-developer-guide.md), however accessing your dev-deployment of
 `openfoodfacts-server` requires an extra step.
 
+## Connect GitHub and Gitpod
+When you use Gitpod, you allow Gitpod to use your GitHub account.
+
+In GitHub, you can review (and revoke if you stop using Gitpod) the access granted to Gitpod: click on your avatar on top right of the screen, then, Settings. In the left panel, under Integrations, click on Applications, then, Authorized OAuth Apps.
+
+In Gitpod side, you can also update what Gitpod is allowed to do with your GitHub account: click on your avatar on top right of the screen, then, Settings. In the left panel, click on Integrations. The line for GitHub should be green. At the end of this line, click on the three dots, then, Edit Permissions. '''If you want to create a pull request via Gitpod, you need to grant public_repo access.'''
```
Comment (diivi) [inline docs/how-to-guides/use-gitpod.md:25]:
```suggestion
On the Gitpod side, you can also update what Gitpod is allowed to do with your GitHub account: click on your avatar on the top right of the screen, then, Settings. In the left panel, click on Integrations. The line for GitHub should be green. At the end of this line, click on the three dots, then Edit Permissions. '''If you want to create a pull request via Gitpod, you need to grant public_repo access.'''
```

File: docs/how-to-guides/use-gitpod.md [hunk 4]
```diff
@@ -40,3 +52,28 @@ on http://openfoodfacts.localhost just as documented in the quickstart guide!
...
+```
+
+If you face some difficulties, you can always look at the logs (use ctrl + c, to quit):  
+```
+$ make log  
+```
+```
+$ make tail  
+```
+At the end of your devs, before to open a pull request, run the following command:
```
Comment (diivi) [inline docs/how-to-guides/use-gitpod.md:76]:
```suggestion
After development, before opening a pull request, run the following command:  
```

**Comment by diivi** [comment]
Nice! I just had some suggestions for grammatical improvements

**Comment by benbenben2** [comment]
> Nice! I just had some suggestions for grammatical improvements

Thank you

**Review by diivi — APPROVED** [review]

**Comment by alexgarel** [comment]
@benbenben2 the prefix for docs PR is "docs:" ! It's important so that release please can build a coherent change log :-)

**Pull request merged**